src/health/azure/himl.py

```diff
@@ -426,6 +441,37 @@ def submit_to_azure_if_needed(  # type: ignore
     exit(0)
 
 
+def _find_file(file_name: str, stop_at_pythonpath: bool = True) -> Optional[Path]:
```

Comment:
do we ever NOT want to stop at pythonpath?

Reply:
People may have entries in PYTHONPATH that reach into a sub directory of their repository, no?